### PR TITLE
Bumped to latest release (2.3.17)

### PR DESCRIPTION
**Pull request by haakobja**

### CODE REVIEW

File: sources [hunk 1]
```diff
@@ -1 +1 @@
-d74e93734cb027b356630faaaf9806bb  inputstream.adaptive-2.2.25.tar.gz
+811634d558d41c0520bbe33099d9b142  inputstream.adaptive-2.3.17.tar.gz
```
Comment (haakobja) [inline sources:1]: Used md5sum to generate the checksum

**Comment by kwizart** [comment]
Thx for the report, please submit a bugzilla.rpmfusion.org so the maintainer can be advertized.
Please also consider joining the co-maintainers for RPM Fusion,
https://rpmfusion.org/Contributors

**Comment by haakobja** [comment]
> Thx for the report, please submit a bugzilla.rpmfusion.org so the maintainer can be advertized.
> Please also consider joining the co-maintainers for RPM Fusion,
> https://rpmfusion.org/Contributors

Done :)

**Comment by leigh123linux** [comment]
https://pkgs.rpmfusion.org/cgit/free/kodi-inputstream-adaptive.git/commit/?id=29273fccbb7aefaa4ef2d5f4ead59cf6e6563b89

**Pull request closed without merging**